WcaOnRails/app/views/competitions/_my_competitions_table.html.erb

```diff
@@ -78,6 +88,11 @@
                 <% end %>
               <% end %>
             </td>
+            <% if bookmarked && competition.registration_not_yet_opened? %>
+              <td>
+                  <%= t('registrations.will_open_html', days: opens_in_days, time: wca_local_time(competition.registration_open)) %>
+              </td>
+            <% end %>
```

Comment:
This entire section feels redundant because `registration_status_icon` includes a mouseover tooltip that specifies when registration opens. So you're stealing a lot of room for information that is already present (albeit somewhat hidden) anyways